### DISCUSSION TITLE
I'm not able to display stats from my private repo even after deploying my own instace

### DISCUSSION THREAD

**Discussion by AleytiaFairlight** [anuraghazra/github-readme-stats#3784] [Q&A] [answered]
The card is in my profile however this is the error
![image](https://github.com/anuraghazra/github-readme-stats/assets/17516560/3b66a86e-e204-4cba-adbb-d360fdcef998)

Here are the permissions for the key
![image](https://github.com/anuraghazra/github-readme-stats/assets/17516560/b031b147-4b89-4352-866a-c3592a663e79)

![image](https://github.com/anuraghazra/github-readme-stats/assets/17516560/e32cb1ab-eb79-4c41-b0c7-41fb5a483062)

![image](https://github.com/anuraghazra/github-readme-stats/assets/17516560/df37d537-da65-459a-8d8f-c44eef50b128)

EDIT 1

Changed my URL just to try and was presented with an error of missing variables

went to settings and found no variables for some reason so I added them

![image](https://github.com/anuraghazra/github-readme-stats/assets/17516560/04741688-69ec-49ff-bd53-8c68dd46d60f)

after redesployment the card still does not work

EDIT 2

here are my domains for testing

Domains
[github-readme-stats-aleytia.vercel.app](https://github-readme-stats-aleytia.vercel.app/)
https://github-readme-stats-aleytiafairlights-projects.vercel.app/
[github-readme-stats-git-master-aleytiafairlights-projects.vercel.app](https://github-readme-stats-git-master-aleytiafairlights-projects.vercel.app/)
[github-readme-stats-l38ajbt17-aleytiafairlights-projects.vercel.app](https://github-readme-stats-l38ajbt17-aleytiafairlights-projects.vercel.app/)

![image](https://github.com/anuraghazra/github-readme-stats/assets/17516560/e24151a7-bd5f-4dd6-9891-1894a8dad4db)

EDIT 3
So as a sanity check cause it might be broken I tried the base URL then tried a public repo of mine

![image](https://github.com/anuraghazra/github-readme-stats/assets/17516560/c1d32241-f58d-4871-8461-31adc21b4b24)

we're back to can't find repo for the private one but the public one works so we know the deployment does work

**Comment by qwerty541** [comment, marked as answer]
Hey @AleytiaFairlight, I just went through the repository data fetcher code and found there these code lines:
https://github.com/anuraghazra/github-readme-stats/blob/1a269d6fa482bf34ad04c6867e2b4fe64e968bae/src/fetchers/repo-fetcher.js#L95-L97
As you see extra pin card is not intended to be used for private repositories and will always show an error.

**Reply by AleytiaFairlight** [reply]
I suppose my next question is if I removed this statement would I be able to display private repo stats?

**Reply by qwerty541** [reply]
I didn't tested, but I think that you would be able to. You can create a separate branch, remove `|| data.user.repository.isPrivate` from that if statement, and deploy it on your own Vercel instance. But I have to warn you that this way everyone will be able to see the card for any of your private repositories, so you do it for your own risk.
